src/libs/API.js

```diff
@@ -52,9 +52,11 @@ function addAuthTokenToParameters(command, parameters) {
         // an API request before we are signed in. In this case, we should just
         // cancel this and all other requests and set isAuthenticating to false.
```

Comment:
One last quick thing, could you please update this comment? I think the part about setting `isAuthenticating ` is no longer accurate.